docs/how-to/generate-blockchain.md

```diff
@@ -1,30 +1,19 @@
 # How to Generate Blockchain
 
 It is possible situation when blockchain is not advanced for more than `k` slots.
-In this case we should generate it. Use `cardano-block-gen` command for it.
+In this case we should generate it. Use `cardano-auxx` and command `generate-blocks` for it.
 
-Basic example of the command:
+Launch `cardano-auxx`:
 
 ```
-$ stack exec -- cardano-block-gen                \
+$ stack exec -- cardano-auxx                     \
     --system-start 0                             \
     --configuration-file node/configuration.yaml \
     --configuration-key mainnet_full             \
-    --blocks 1000                                \
-    --generated-db some-db                       \
-    --secret PATH_TO_SECRET_KEY                  \
-    --tx-count "(0,0)"                           \
-    --append
+    --db-path some-db                            \
 ```
```

Comment:
What is it? AFAIK this command as-is won't work. You need to specify a list of command or pass `repl`.